src/classes/DatatableController.cls

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by Synebo on 13.10.2021.
+ */
+
+public with sharing class DatatableController {
+
+    @AuraEnabled(Cacheable=true)
+    public static List<Wrapper> getContacts(String accountId) {
+        List<Contact> contactsList = new List<Contact>();
+        List<Wrapper> wrappers = new List<Wrapper>();
+        if(accountId == null) {
```

Comment:
Better to use one place to specify query as if list of fields will be changed, you will have to change it in two places
```
String query = 'SELECT Id, Name, Phone, Title,AccountId, Account.Name, Account.Industry, Account.Rating FROM Contact';
if(accountId != null) {
        query += ' WHERE AccountId = :accountId';
}
contactsList = Database.query(query);
```